src/metabase/api/activity.clj

```diff
@@ -86,19 +90,33 @@
                            (hydrate :user :table :database)
                            add-model-exists-info)))
 
+(defn- models-query
+  [model ids]
+  (db/select
+      (case model
+        "card"      [Card
+                     (db/qualify Card :id) :name :collection_id :description :display
+                     :dataset_query :dataset :archived
+                     (db/qualify Collection :authority_level)]
+        "dashboard" [Dashboard
+                     (db/qualify Dashboard :id) :name :collection_id :description
+                     :archived
+                     (db/qualify Collection :authority_level)]
+        "table"     [Table
+                     :id :name :db_id
+                     :display_name :initial_sync_status
+                     :visibility_type])
+                    (cond-> {:where [:in (db/qualify (symbol (str/capitalize model)) :id) ids]}
+                      (not= model "table")
```

Comment:
`(not= model "table')` is inside of the clause when model is equal to table.

Reply:
ah. the indentation here is off so it isn't clear that the `cond->` is the argument to the `db/select`

Reply:
you don't need the `(db/qualify Dashboard :id)` or `(db/qualify Card :id)` (and note you don't have it on the `Table`'s id.